docker/pravega/Dockerfile

```diff
@@ -9,6 +9,9 @@
 #
 FROM java:8
 
+RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
```

Comment:
@vroy I don't fully follow this comment:

> This will always be ubuntu as the image starts off with an ubuntu image (FROM java:8)

I'm not sure why you say it will always be ubuntu. If we are doing this for docker, then shouldn't I be able to run it on any OS that can install docker? Is there a way of not limiting it to ubuntu?